### PR TITLE
MBL-12818 - hide rubric points when setting is true

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-12818
affects: Student
release note: none

### CODE REVIEW

File: Core/Core/Models/Assignment.swift [hunk 1]
```diff
@@ -107,6 +108,10 @@ extension Assignment {
             }
         }
 
+        if let apiRubricSettings = item.rubric_settings {
+            hideRubricPoints = apiRubricSettings.hide_points
+        }
```
Comment (armstrongnate) [inline Core/Core/Models/Assignment.swift]: nitpick: this could be `hideRubricPoints = item.rubric_settings?.hide_points == true`

**Comment by armstrongnate** [comment]
I will QA once it builds.

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student

[MBL-12818](https://instructure.atlassian.net/browse/MBL-12818)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 95.35% | 95.36% | -0.01%
**Student** | 90.78% | 90.78% | 0.01%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.78% | 92.78% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by armstrongnate** [comment]
Garrett, here's that rubric from the ticket with large font turned up a little bit:

IMG_5AC3DEB7416C-1

Dave says that we don't want the circles to have a max width- they should only truncate once it's maxed out the width of the phone. He says that in the screenshot above the `a...` circle should have dropped to the next line and not truncated.

Dave also says there needs to be more left and right padding inside the circle.

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift [hunk 2]
```diff
@@ -167,7 +179,8 @@ class RubricCircleView: UIView {
 
     func updateAccesibilityOnSelectedButton(previous: UIButton?, next: UIButton?) {
         var defaultSelectedButton: UIButton?
-        if let defaultSelectedIndex = self.rubric?.selectedIndex {
+        if var defaultSelectedIndex = self.rubric?.selectedIndex {
+            if rubric?.isCustomAssessment ?? false { defaultSelectedIndex = self.buttons.count - 1 }
```
[the remark this answers is not in the document]
Comment (bluwave) [reply Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: good suggestion.

**Pull request merged**